src/main/java/org/fenixedu/academicextensions/services/registrationhistory/RegistrationHistoryReport.java

```diff
@@ -1106,4 +1105,20 @@ public String getHealthCardNumber() {
         return getPerson().getHealthCardNumber();
     }
 
+    public String getFinalConclusionNumber() {
```

Comment:
For consistency RegistrationHistoryReport should not declare this method, since it already returns partial and final conclusion report objects, which are null safe (they return empty when no information is available), and we are not expanding other methods from conclusion into this bean (e.g. rawGrade).

UI should use conclusionReport.conclusionNumber directly.